charts/istio-authorizer/values.yaml

```diff
@@ -103,6 +108,12 @@ clientCredentials:
   clientID: "YnZnN2h2YzA1NnV0YmdpaHF1OWcK"
   clientSecret: "aE0wdmYwZHNBMHkxSHBHZW1WXzlDa21iYmlOclp2NW92S1JDTm5hd2pJbwo="
 
+# Authorizer discovery configuration
+discovery:
+  # List of namespaces in which authorizer will do service discovery
+  namespaces:
+    - default
```

Comment:
Can we add more here just for demonstration purposes that you add any additional namespaces with `- example` and not, for example, with `- default, example`?